frontend/src/index.pug

```diff
@@ -37,10 +37,15 @@ html
           .timestamp-footer
             span Generated by &nbsp;
             a(
-              v-bind:href="getRepoSenseLink()", target="_blank",
+              v-bind:href="getRepoSenseMainSiteLink()", target="_blank",
             )
-              strong RepoSense {{ repoSenseVersion }}
-            span &nbsp; on {{ creationDate }}
+              strong RepoSense
+            span  (
```

Comment:
Two space here, one space is enough right?

Reply:
Umm, actually so the way this works is if there is a single space then using "span" just prints it. So I had to give to two space so that an actual space comes before the bracket.

Reply:
But I just realised there is an extra space after the last bracket, so will correct that and push again. 

![Screen Shot 2019-09-16 at 11 51 04 PM](https://user-images.githubusercontent.com/35946746/64973089-fb2dd380-d8dc-11e9-92cd-1952e4e0df07.png)

Reply:
We should standardize to use `&nbsp;` for such intended spaces
E.g. double spaces, spaces at end of the line should be replaced with `&nbsp;` to highlight its presence

Reply:
> We should standardize to use `&nbsp;` for such intended spaces
> E.g. double spaces, spaces at end of the line should be replaced with `&nbsp;` to highlight its presence

Oh okay. Didn't know we were using `&nbsp;` for this purpose. Thank you for guiding.